S21/rebelDi/tests/resources/tests.tsv

```diff
@@ -0,0 +1,36 @@
+HELLO	en	hello
+WORLD	en-US	world
+cAmEl	en-IE	camel
+---OK	en-Latn	---ok
+tAcht	ga	t-acht
+tACHT	ga	t-acht
+TACHT	ga	tacht
+nAthair	ga	n-athair
+nATHAIR	ga	n-athair
+NATHAIR	ga	nathair
+nÓg	ga-IE	n-óg
+KASIM	tr	kasım
+KASIM	en	kasim
+ΠΌΛΗΣ	el	πόλης
+ΣΠΌΛΗ	el	σπόλη
+官话	zh-Hans	官话
+ภาษาไทย	th	ภาษาไทย
+ПрИвет	ru	привет
+МАМА	ru	мама
+ConSEQUences	en	consequences
+TəkRarLAMa	az	təkrarlama
+DÜNYA	az	dünya
+QALIB	az	qalıb
+Ονομα	el	ονομα
+ΣΥΓΧΑΡΗΤΗΡΙΑ	el	συγχαρητηρια
```

Comment:
I'm not sure if I have missed how greek words are converted. I would like to see how greek words are handled.

Reply:
There are several examples of Greek in the file, I'll just try to combine them, so it would be more readable.